src/components/cc-global-header.vue

```diff
@@ -96,11 +96,43 @@ if (process.env.NODE_ENV === "development") {
 } else {
   var axios = window.axios;
 }
+function retryApiRequest(error) {
+  let config = error.config;
+  // If config does not exist or the retry option is not set, reject
+  if (!config || !config.retry) return Promise.reject(error);
+
+  // Set the variable for keeping track of the retry count
+  config.retryCount = config.retryCount || 0;
+
+  // Check if we've maxed out the total number of retries
+  if (config.retryCount >= config.retry) {
+    // Reject with the errors
+    return Promise.reject(error);
+  }
+
+  // Set the default retry delay in milliseconds
+  const defaultRetryDelay = 1000;
+
+  // Increase the retry count
+  config.retryCount += 1;
+
+  // Create new promise to resolve the request
+  const requestResolver = new Promise(function (resolve) {
+    setTimeout(function () {
+      resolve();
+    }, config.retryDelay || defaultRetryDelay);
+  });
+
+  // Return the promise which recalls axios to retry the request
+  return requestResolver.then(function () {
+    return axios(config);
+  });
+}
 export default defineComponent({
   name: "cc-golbal-header",
   created() {
     var vm = this;
-    var requestPath = "/?rest_route=/ccnavigation-header/menu";
+    var requestPath = "/?rest_route=/ccnavigation-header/men";
```

Comment:
Is this a typo?

Reply:
It's how I was testing the failed API calls by triggering a 404 Not Found. I already corrected it in the latest commit.